src/js/index.js

```diff
@@ -0,0 +1,72 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const home = document.getElementById('sliderHome')
+  const services = document.getElementById('services')
+  const portfolio = document.getElementById('portfolio')
+  const about = document.getElementById('aboutPortfolio')
+  const contact = document.getElementById('contact')
+  const navbarPortfolio = document.getElementById('navbarPortfolio')
+  const navbarHome = document.getElementById('navbarHome')
+  const navbarservices = document.getElementById('navbarservices')
+  const navbarAbout = document.getElementById('navbarAbout')
+  const navbarContact = document.getElementById('navbarContact')
+  const headerHeight = document.querySelector('.wrapper__header').getBoundingClientRect().height
+  const scrollOptions = {
+    behavior: 'smooth',
+    block: 'start',
+  }
+
+  document.addEventListener('scroll', () => {
+    let scrolled = window.pageYOffset || document.documentElement.scrollTop
+    const arry = [
```

Comment:
What does the word "arry" mean? Do you mean "array"?
And the second thing about it:  every time user scrolls new array will be created. But as far as I understand, you don't need to create it every time. You need to create it only ONCE. So, replace this  constant outside addEventListener.